subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/intra_vc_scheduler.go

```diff
@@ -28,22 +28,42 @@ import (
 	"k8s.io/klog"
 )
 
-// intraVCScheduler is an interface for allocating cells inside a VC.
-// It stores two maps of ChainCellList, one for reserved cells, the other
-// for non-reserved ones. It should be able to return a virtual cell
-// on receiving a cell request.
+// intraVCScheduler is an interface for scheduling pods inside a VC.
+// It stores two maps of ChainCellList, one for reserved cells, the other for non-reserved ones.
+// It should be able to return a set of GPU placements in the VC for a scheduling request.
 type intraVCScheduler interface {
 	getNonReservedCellList() map[CellChain]ChainCellList
 	getReservedCellList() map[api.ReservationId]ChainCellList
 
-	// Cell allocation inside a VC. We use buddy alloc for default
-	// (see defaultIntraVCScheduler).
-	allocateCell(CellRequest) *VirtualCell
+	// Scheduling a series of pods inside a VC. We use topologyAwareScheduler by default.
```

Comment:
>a series of pods [](start = 15, length = 16)

The "series of pods" must be the same as "pods in the same affinity group", if so why not directly use latter?